settings/DevHome.Settings/Views/PreferencesPage.xaml.cs

```diff
@@ -44,4 +45,14 @@ private void BreadcrumbBar_ItemClicked(BreadcrumbBar sender, BreadcrumbBarItemCl
             crumb.NavigateTo();
         }
     }
+
+    private async void Page_Loaded(object sender, RoutedEventArgs e)

```

Comment:
I'd recommend making the `OnPageLaunched` method a command instead (eg. maybe call it "InitializeAsync" or something), and just have an `EventTriggerBehavior` in XAML binding to that command from the `Loaded` event. It'll let you remove this code behind entirely 🙂